WordPress/src/main/java/org/wordpress/android/ui/compose/components/ResourceImage.kt

```diff
@@ -0,0 +1,32 @@
+package org.wordpress.android.ui.compose.components
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+
+/**
+ * Image component used to display an image from app resources.
+ */
+@Suppress("LongParameterList")
+@Composable
+fun ResourceImage(
```

Comment:
NIT: I would've personally inlined this.

Reply:
Sounds good to me, applied this in commit https://github.com/wordpress-mobile/WordPress-Android/pull/16934/commits/55587bae269d43bd99c8f4fe66fa04873c63c2fb.